Dockerfile

```diff
@@ -6,7 +6,16 @@ COPY ./src ./src
 
```

Comment:
Is this image the one being used in our dev cluster currently?

Reply:
It is, yes.